<!DOCTYPE html>
<html>
<head>
    <meta name="description" content="Selective Vulnerability of Neurodegenerative Diseases">
    <meta name="keywords" content="tagger, NGS, next generation sequencing, transgenic mice, RiboTag, Ribo-Tag, Argonaute2, Ago2, Huntington disease, Prion disease, neurodegeneration, RNAseq, Cas9, transcriptomics">
    <meta name="author" content="Lech Kaczmarczyk">
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
 </head>

<body>
<h4>Interactive web app for browsing the RML Ribotag data</h4>
<p>There are 2 parts - tabs in the Navigation Bar:</p>
<p><strong>Single gene look-up</strong></p>
<p>Selecting a gene by symbol or Ensembl ID generates (top) a bar plot of TPMs in control (NBH, normal brain homogenate) 10 weeks timepoint samples, and (bottom) a bar plot of log2 fold enrichment between RML and control, for each cell type, both timepoints. You can toggle between individual (free) scaling and fixed scaling for the y-axis.</p>
<p><strong>Dot</strong> <strong>plot</strong><br/>The dot plot shows each individual gene as a single point. Data are separated into cell types and timepoints. Points can be selected by clicking or dragging a selection box (move the cursor while holding the left mouse button). Selected genes are highlighted, labeled, and listed in the table below the plot, which can be printed or exported in multiple formats using the widget above the table. The 'Reset' button in the side panel de-selects all highlighted genes. Genes can also be de-selected by clicking and dragging (dragging across a mix of selected and not-selected points will invert the selection). The slider in the side panel restricts the displayed genes based on FDR threshold (BH).</p>
</body>
</html>
